server/sonar-web/src/main/js/app/types.ts

```diff
@@ -105,7 +105,7 @@ export interface Metric {
   domain?: string;
   hidden?: boolean;
   key: string;
-  name: string;
+  name?: string;
```

Comment:
metric without name?

Reply:
It was an easy way of fixing painful problem, can we discuss it after lunch ?